vars/mavenCI.groovy

```diff
@@ -63,6 +63,7 @@ def call(body) {
 
         stage('Integration Testing') {
             mavenIntegrationTest {
+                integrationTestCmd = config.integrationTestCmd
```

Comment:
this should just be called `cmd` instead of `integrationTestCmd` since we have already established that the current context is `mavenIntegrationTest`